libbindgen/src/ir/comp.rs

```diff
@@ -954,6 +954,10 @@ impl TypeCollector for CompInfo {
             types.insert(var);
         }
 
-        // FIXME(emilio): Methods, VTable?
+        for method in self.methods() {
+            types.insert(method.signature);
+        }
+
+        // FIXME(emilio): VTable?
```

Comment:
Yeah, I think every type that could potentially be used in the vtable would be caught by the bases or the methods, so you should be able to remove this comment.